src/globals/scss/_theme-tokens.scss

```diff
@@ -261,27 +130,23 @@ $carbon--theme--v9: (
 
   // Skeleton Loading
   /// @access public
-  $skeleton: rgba($color__blue-51, 0.1) !default !global; //old $field-01 TODO: Define for experimental
+  $skeleton: rgba(#3d70b2, 0.1) !default !global; //old $color__blue-51/$field-01 TODO: Define for experimental
 
   // ☠️ Deprecated
   /// @deprecated
   /// @access public
   $hover-field: $hover-ui !default !global;
   $active-01: $active-ui !default !global;
-  $nav-01: $color__navy-gray-1 !default !global;
-  $nav-02: $color__blue-90 !default !global;
-  $nav-03: $color__purple-30 !default !global;
-  $nav-04: $color__purple-60 !default !global;
-  $nav-05: $color__teal-40 !default !global;
-  $nav-06: $color__teal-50 !default !global;
-  $nav-07: $color__blue-30 !default !global;
-  $nav-08: $color__blue-51 !default !global;
+  $nav-01: #0f212e !default !global; // Old $color__navy-gray-1
```

Comment:
This PR is to remove v9 styles. Does that also include items that had been deprecated like `$nav-01`? Searching `@deprecated` revealed a number of deprecated items in our source still.

Reply:
Thank you for bringing this up - I didn't remove as `v10` code still has reference to `$hover-field`, etc. Added this to "non-removed" list in the issue description. Will bring this up as a discussion and will remove it in a separate PR if our consensus is to remove.